cli/command/stack/swarm/services.go

```diff
@@ -10,10 +10,12 @@ import (
 )
 
 // GetServices is the swarm implementation of listing stack services
-func GetServices(ctx context.Context, dockerCli command.Cli, opts options.Services) ([]swarm.Service, error) {
+//
+// Deprecated: this function was for internal use and will be removed in the next release.
+func GetServices(ctx context.Context, dockerCLI command.Cli, opts options.Services) ([]swarm.Service, error) {
 	var (
-		err    error
-		client = dockerCli.Client()
+		err       error
+		apiClient = dockerCLI.Client()
```

Comment:
Minor conflict here, because these vars were renamed in https://github.com/docker/cli/commit/7118f1fb4b7e44d1b3f7b829b822a21a128eac30, which isn't in this branch; I chose to just apply the same renames as part of this cherry-pick.